src/index.ts

```diff
@@ -48,6 +48,7 @@ export enum CreateAccountError {
   reserved_field,
 
   id_must_not_be_zero,
+  id_must_not_be_max_int,
```

Comment:
```suggestion
  id_must_not_be_int_max,
```

We should change this upstream as well—would you please do that too @koekiebox ?

Reply:
@jorangreef  Done. https://github.com/coilhq/tigerbeetle/pull/131